### DISCUSSION TITLE
gradle plugin for managing all checkstyle settings in a project

### DISCUSSION THREAD

**Discussion by jgitik** [checkstyle/checkstyle#12977] [Q&A]
Hello. I would like to write or use a gradle plugin that could transmit to all consumers:
 1)the same version of checkstyle,
 2) a file with settings 
 3) the path to this file.
   How can I do this? Is there a plugin already implemented somewhere

**Comment by romani** [comment]
there is no such plugin, I am not aware about this possibilities at all.
